airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/python_connectors.py

```diff
@@ -215,7 +215,14 @@ async def _run(self, connector_under_test: Container) -> StepResult:
 
         test_environment = await self.install_testing_environment(with_poetry(self.context))
         test_execution = test_environment.with_(
-            hacks.never_fail_exec(["airbyte-lib-validate-source", "--connector-dir", ".", "--validate-install-only"])
+            hacks.never_fail_exec(
+                [
+                    "pyab",
+                    "validate",
+                    f"--connector={self.context.connector.technical_name}",
+                    "--pip-url='.'",
```

Comment:
What's the meaning of `--pip-url='.'` ?

Reply:
Pip URL of "." just means to install with `pip install ." - meaning install the Python package from the current working directory. 

This is meant to keep parity with the previous validate command while also supporting other install sources, such as a named PyPi reference or a versioned PyPi reference, or a guy URL, etc